docs/conversion_examples_gallery/behavior/movie.rst

```diff
@@ -0,0 +1,37 @@
+Video data conversion (multimedia formats)
+------------------------------------------
+
+Install NeuroConv with the additional dependencies necessary for reading multimedia data.
+
+.. code-block:: bash
+
+    pip install neuroconv[movie]
+
+This interface can handle conversions from avi, mov, mp4, wmv, flv and most FFmpeg_ supported formats to NWB using the
+:py:class:`~neuroconv.datainterfaces.behavior.movie.moviedatainterface.MovieInterface` class.
+
+.. _FFmpeg: https://ffmpeg.org/
```

Comment:
Can you move this to the https://github.com/catalystneuro/neuroconv/blob/add_movie_interface_gallery/docs/conf.py#L90 area so all external links are in one place for easy searching/fixing? (tomorrow, I know you're done for today)

Reply:
I tried this for like 15 minutes and I could not figure out how to make it work. The documentation was not clear to me either:
https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html

I tried:
```

extlinks = {
    "nwbinspector": ("https://nwbinspector.readthedocs.io/en/dev/%s", ""),
    "FFmpeg": ("https://ffmpeg.org/%s", ""),
}

```
With the following signature:
```
:FFmpeg:`FFmpeg`
```

But this generates this:
![image](https://user-images.githubusercontent.com/6429509/196976605-5c2d8237-cc14-45a4-bd75-8a7987c3d50b.png)

Reply:
Just need the string to be :FFmpepg:`FFmpeg <>`.

Example on the Inspector: https://github.com/NeurodataWithoutBorders/nwbinspector/blame/dev/docs/best_practices/nwbfile_metadata.rst#L88 using https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/docs/conf_extlinks.py#L26

Reply:
I tried that and it showed the `<>` on the document.